src/app/tables/table.component.ts

```diff
@@ -165,7 +170,11 @@ export class EstuaryTableComponent implements OnChanges {
     const tempLink = document.createElement('a');
     tempLink.href = 'data:text/csv;charset=utf-8,' + encodeURI(csvContent);
     tempLink.target = '_blank';
-    tempLink.download = this.title.toLowerCase().replace(/ /g, '_') + '.csv';
+    if (this.title) {
+      tempLink.download = this.title.toLowerCase().replace(/ /g, '_') + '.csv';
+    } else {
+      tempLink.download = this.csvName.toLowerCase().replace(/ /g, '_') + '.csv';
```

Comment:
Since you are passing `csvName` directly, I think it makes sense for `csvName` to be the file name it will be called. For example, you'd pass something like `recent_artifacts.csv`.

Reply:
@mprahl I actually think it makes more sense to not include the .csv extension when passing in csvName, or, alternatively, accept both variations and check to see if it already has '.csv' on the end of it.

Reply:
Sure, I like that approach of add `.csv` if it's not provided.